### ISSUE TITLE
New way of including images in proposal directory adds images twice to HTML site

### ISSUE THREAD

**Issue by henrik-berglund** [ProAgileAB/agile-team-development#25]
![image](https://user-images.githubusercontent.com/1319563/49344480-05495300-f678-11e8-8802-7afdc1864f97.png)

**Comment by e1th0r** [comment]
Yes, extra markup is working on pdf files without problem but adding extra image on html files...

**Comment by henrik-berglund** [comment]
I think this should be quite easy to fix by having a small script on the pdf generation side (PC), that changes img tags to markdown.
